### PR TITLE
Dev guide release post

### PR DESCRIPTION
**Pull request by maelle**
Direct link to preview https://deploy-preview-53--ropensci.netlify.app/blog/2020/10/22/devguide-0.5.0/

* [x] I have read the Content Guidelines.
* [x] I have read the Technical Guidelines.
* [x] I used or followed the R Markdown or Markdown template.
* [x] I have followed the Style Guide.
* [x] I created or updated my author metadata with correct folder name.
* [x] I have added relevant tags after browsing existing tags.
* [x] I ran `roblog::ro_lint_md()` on index.md (optional).
* [x] I ran `roblog::ro_check_urls()` on index.md (optional).
* [x] I ran a spell-check on index.md.

### CODE REVIEW

**Comment by maelle** [comment]
Note that some URLs will remain broken until I actually merge the dev branch of the dev guide into the default branch.

**Review by steffilazerte — COMMENTED** [review]
Hi @maelle!
Thanks for the shout-out :hugs: ! This is an informative and concise update post. On the whole everything looks fine. I have a couple (5) suggestions for clarifying wording or fixing typos, but nothing major. Thanks!

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [X] publication date is ok
* [X] alternative text of images is informative
* [X] Twitter metadata looks ok (paste post preview link in [Twitter card validator](https://cards-dev.twitter.com/validator); might have to click twice on "Preview card")
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok

**Comment by stefaniebutland** [comment]
Regarding conflicting files noted above, the live versions of Lincoln's and Mauro's (merged https://github.com/ropensci/roweb3/pull/63) author pages are the correct ones.

**Comment by stefaniebutland** [comment]
@maelle Thank you for the lovely Contributing Guide shoutout 💚

File: content/author/julia-gustavsen/_index.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Julia Gustavsen
+link: https://www.juliagustavsen.com/
+twitter: JuliaGustavsen
+github: jooolia
```
Comment (stefaniebutland) [inline content/author/julia-gustavsen/_index.md:5]:
```suggestion
github: jooolia
orcid: 0000-0002-4764-4802
img: img/team/julia-gustavsen.jpg
```

Comment (stefaniebutland) [reply content/author/julia-gustavsen/_index.md:5]: Julia requested these additions in her author page.

**Comment by maelle** [comment]
Thanks!

I'll update this thread on Thursday once I have released the dev guide, please don't merge before that. :pray:

**Comment by maelle** [comment]
Reg a tweet maybe something simple like 

```
[blog] rOpenSci Dev Guide 0.5.0: Updates

A highlight of the recent changes is the addition of a package curation policy! 📦

Read more: <link>

Thanks a lot to all contributors! 🙏
```

**Comment by steffilazerte** [comment]
What do you think?

```
[blog] rOpenSci Dev Guide 0.5.0: Updates

In particular highlighting the addition of a package curation policy! 📦

Read more: https://ropensci.org/blog/2020/10/22/devguide-0.5.0/

Thanks a lot to all contributors! 🙏
```

**Comment by maelle** [comment]
awesome!

Btw is it easier if I merge myself after the release? 🤔

**Comment by steffilazerte** [comment]
I'm not sure it matters, I'll have to send out the tweet and I definitely don't want to tweet before it's posted :D Either way, give me a ping :D

**Comment by maelle** [comment]
I've now released the guide so it's ready for the post. Thanks @steffilazerte!

**Pull request merged**